### DISCUSSION TITLE
Two RED errors when updating Oobabooga via update_windows.bat "ERROR: pip's dependency resolver does not currently "

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#2641] [General]
Error #1 
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.3 which is incompatible.
Successfully installed accelerate-0.20.3 datasets-2.12.0 numpy-1.24.3 safetensors-0.3.1 transformers-4.30.0

Error #2
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
auto-gptq 0.2.2+cu117 requires accelerate>=0.19.0, but you have accelerate 0.18.0 which is incompatible.
Successfully installed accelerate-0.18.0 datasets-2.10.1 safetensors-0.3.0 transformers-4.28.0

Are these actual errors that I should work on resolving? If so any advice on how to properly resolve? 
Thanks.

**Comment by BetaDoggo** [comment]
If the webui still starts and there aren't any other issues then these errors aren't worth dealing with. If there are issues you could try reinstalling all of the pip dependencies and hoping that it gets it right this time.

**Reply by viperwasp** [reply]
Thanks so far I don't have any issues except Blas won't work. I can't use GGML with GPU acceleration. I don't think these errors are the problem as I think I manually fixed them and it did not work. But the errors come back when I update. But I could be wrong I may not have fixed them properly in the first place. But if these errors are not causing my GPU accleration issue than they are fine.

**Reply by viperwasp** [reply]
Errors went away with fresh reinstall of Ooba. Likely was other ways to fix this but I installed Ooba anyways to to fix something else and errors vanished.
